src/main/resources/kibana-rest.yml

```
@@ -72,6 +79,7 @@ kibana_detection_engine_rules_prebuilt_status:
     ">= 7.10.0 < 7.15.0": "/api/detection_engine/prepackaged"
     ">= 7.15.0 < 8.9.0": "/api/detection_engine/rules/prepackaged/_status"
     ">= 8.9.0": "/internal/detection_engine/prebuilt_rules/status"
+    # ">= 9.0.0": "uri [/internal/detection_engine/prebuilt_rules/status] with method [get] exists but is not available with the current configuration"
```

Does this mean we should add `< 9.0.0` to the above line to disable it?

```suggestion
    ">= 8.9.0 < 9.0.0": "/internal/detection_engine/prebuilt_rules/status"
    # ">= 9.0.0": "uri [/internal/detection_engine/prebuilt_rules/status] with method [get] exists but is not available with the current configuration"
```